tests/EdFi.Admin.DataAccess.IntegrationTests/appsettings.json

```diff
@@ -1,8 +1,9 @@
 {
   "ApiSettings": {
-    "Engine": "SQLServer"
+    "Engine": ""
   },
   "ConnectionStrings": {
-    "EdFi_Admin": "Server=(local); Database=EdFi_Admin_Test; Trusted_Connection=True; Application Name=EdFi.Admin.DataAccess.IntegrationTests"
+    "MSSQL": "Server=(local); Database=EdFi_Admin; Trusted_Connection=True; Persist Security Info=True; Application Name=EdFi.Admin.DataAccess.IntegrationTests; Encrypt=False;",
+    "PostgreSQL": "Host=localhost;Port=5432;username=postgres;database=EdFi_Admin;Application Name=EdFi.Admin.DataAccess.IntegrationTests;"
```

Comment:
Are these changes needed? You could revert the file changes and get the _EdFi_Admin_ connection string with: `config.GetConnectionString("EdFi_Admin");`
Because initdev automatically initializes `appsettings.Development.json` with the appropriate engine and connection 
 string.
Also, I believe we should use the `_Test` databases (`EdFi_Admin_Test`, for example); which initdev already initializes in `appsettings.Development.json`.